### DISCUSSION TITLE
Follow up demographic data missing (?)

### DISCUSSION THREAD

**Discussion by rouzbehrahai** [now-i-know-my-abcd/docs#48] [Stats & Code Help] [answered]
Hi All, 

I notice in the protocols that multi-year follow ups were conducted with demographics data. But when I download the data much of data on follow up years is missing (age, income, etc.). Why might this be? This is regarding 5.0 release. 

Thanks,

**Comment by mangstad** [comment, marked as answer]
Many of the demographics are stored under slightly different variable names for visits after baseline for some reason. For example, in the abcd_p_demo file for Release 5.0, there are variables demo_comb_income_v2 and demo_comb_income_v2_l. The first only contains baseline data, with the follow-up visits under the _l variable. Many other demographics are arranged similarly.

**Reply by rouzbehrahai** [reply]
Thanks so much Mike. My response is a bit late, but wanted to say this is very helpful.
